DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: binance
 Type: Package
 Title: Wrapper for Binance API
-Version: 0.0.1
+Version: 0.0.3
```

Comment:
Please don't bump the version. I'll do that shortly.

Reply:
Reverted on next commit